odbc/src/api/odbc_uri.rs

```diff
@@ -1,41 +1,129 @@
 use crate::errors::{ODBCError, Result};
+use lazy_static::lazy_static;
+use regex::{RegexSet, RegexSetBuilder};
 use std::collections::HashMap;
 
-// TODO SQL-990: These errors will probably change.
-const NOT_EMPTY_ERROR: &str = "uri must not be empty";
-const EQUAL_ERROR: &str = "all uri atttributes must be of the form key=value";
+const EMPTY_URI_ERROR: &str = "URI must not be empty";
+const EQUAL_ERROR: &str = "all URI attributes must be of the form keyword=value";
```

Comment:
```suggestion
const INVALID_ATTR_FORMAT_ERROR: &str = "all URI attributes must be of the form keyword=value";
```
A rename for this error could be useful, too.